addon/ajax-request.js

```diff
@@ -200,8 +200,8 @@ export default class AjaxRequest {
    */
   _getFullHeadersHash(headers) {
     const classHeaders = get(this, 'headers') || {};
-    const _headers = merge({}, classHeaders);
-    return merge(_headers, headers);
+    const _headers = assign({}, classHeaders);
```

Comment:
`assign` allows more than two arguments so we should be able to do this in a single invocation:

```js
return assign({}, classHeaders, headers);
```